.github/workflows/testing.yml

```diff
@@ -38,26 +38,28 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ "3.7", "3.8" , "3.9", "3.10" ]
-        django-version: [ "3.0", "3.1", "3.2", "4.0" ]
+        python-version: [ "3.7.13", "3.8.13" , "3.9.13", "3.10.6", "3.11.0-rc.1" ]
```

Comment:
NIP: I think we can go with specifying just the major and minor components of the Python version, so `setup-python` will install the latest Python patch version. Going with this approach (so specifying also patch piece of Python version) we will have to remember to update the `testing.yml` workflow matrix every time a new python patch version will be released.